### ISSUE TITLE
Confused about where code needs to go

### ISSUE THREAD

**Issue by remirr** [twrecked/hass-momentary#16]
So I added:
momentary: 
to my config.yaml

But where do I add the following? Because I added it to a manual card on the lovelace dashboard but it doesn't seem to work. So I have an input select in my Automations.yaml called: input_select.spa_first_blower. Which is linked to the entity called: pool1_channels2_mode in configuration.yaml

So how do I link that to your card? 

Where do I put the following and how to I link it to the Entity or the Input_Select, which in addition, which one do I link it to? 

In summary: my API to my SPA cycles through states, 'auto' 'on' 'off'. it does not allow me to select one using input select, I actually need to cycle. So I am trying to create a button which allows me to cycle through and shows the current status on the dashboard so that after we finish in the spa we can set it back to 'off'

switch:
  - platform: momentary
    name: Empty House Trigger
    mode: on
    toggle_for: 5
    cancellable: True

**Comment by twrecked** [comment]
So, you just need a button you press to cycle through the SPA.

Can you paste the automation you have at the moment?

Can you tell me the integration type of the SPA?

**Comment by remirr** [comment]
Thanks for your response. 

I have added the requested info below. The communication is via REST Api with their systems. The problem I am having with the below is that input_select with a drop down does not work because the API does not allow you to select a 'state' it only allows you to cycle through the individual modes - off, on, auto. Ideally I would like to have a button that just sets the state but I am not sure how I would even do that. 

So my backup was just to have a button that says "Top Spa Bubbles" and the current state that it is in, so you can press it - check the state and press it again and check the new state. 

Thank you for your help!

In my config.yaml I have 

  spa_first_blower:
    name: Top Spa Bubbles
    options:
      - "Auto"
      - "On"
      - "Off"
In my automation.yaml I have:
- id: '1628215649697'
  alias: Top Spa Bubbles On
  description: ''
  trigger:
  - platform: state
    entity_id: input_select.spa_first_blower
    to: 'on'
  condition: []
  action:
  - service: rest_command.poolaction
    data:
      action_code: 1
      device_number: 2
      value: 2
  - delay: 2
  - service: homeassistant.update_entity
    entity_id: sensor.pool1_channels2_mode
  mode: single
- id: '1628215649696'
  alias: Top Spa Bubbles Off
  description: ''
  trigger:
  - platform: state
    entity_id: input_select.spa_first_blower
    to: 'off'
  condition: []
  action:
  - service: rest_command.poolaction
    data:
      action_code: 1
      device_number: 2
      value: 0
  - delay: 2
  - service: homeassistant.update_entity
    entity_id: sensor.pool1_channels2_mode
  mode: single

**Comment by BottoH** [comment]
Hello twrecked. 
First of all "Thank you for your work".
I´m a noob at _Home Assistant_ which currently runs on my rasp pi 3b.
I already got _KNX_ integration running and I´m able to interact with the bus.
Now I´d like to have the _momentary_ switch behaviour for my KNX switches (light, sunshade etc)
Why would I like to have this?! Well, my KNX components are communicating with a _WAGO SPS_ and therefor the KNX sensors all act like push buttons. It is just the way I implemented it and I feel quit comfortable with this. 
So the KNX group adresse (i. e. 0/0/1) will have the state ON (1) for a rising edge an OFF (0) for the falling edge - when the button is released.
I would like to have the same behaviour for my visu and hopefully the momentary thing will do the trick.

I installed your software via HACS and edited the `cofiguration.yaml` like it is described in your doc.
I already have the following package code running for KNX entities:

/config/Packages/knx.yaml:
```
######################
# KNX_IP_ITF
######################
knx:
  tunneling:
    host: "192.168.0.254"
######################
# Lights
######################
  switch:
    name: "Test Light"
    address: "0/0/1"
   state_address: "0/0/2" //optional
```

Now where do I add the momentary behaviour to the switches?
I guess I´ve to do this in the _lovelace raw configuration_ but I just don't get it... sorry :)

The next step would be to add `state_address` to the KNX switch so that the final button card will send On OFF and also be able to show the state of the output.

I hope you are willing to help me with this (little) problem?!
Regards
BottoH
